Documentation/format/index.md

```diff
@@ -0,0 +1,205 @@
+```
+
+
+### Series
+
+The section contains a sequence of series that hold the label set of the series as well as its chunks within the block. The series are sorted lexicographically by their label sets.  
+The file offset to the beginning of a series serves as the series' ID in all subsequent references. Thereby, a sorted list of series IDs implies a lexicographically sorted list of series label sets.
+
+```
+┌───────────────────────────────────────┐
+│  #series <4b>                         │
+├───────────────────────────────────────┤
+│ ┌───────────────────────────────────┐ │
+│ │   series_1                        │ │
+│ ├───────────────────────────────────┤ │
+│ │                 . . .             │ │
+│ ├───────────────────────────────────┤ │
+│ │   series_n                        │ │
+│ └───────────────────────────────────┘ │
+└───────────────────────────────────────┘
+```
+
+Every series entry first holds its number of labels, followed by tuples of symbol table references that resemble label name and value. The label pairs are lexicographically sorted.  
+After the labels, the number of indexed chunks is encoded, followed by a sequence of metadata entries containing the chunks minimum and maximum timestamp and a reference to its position in the chunk file. Holding the time range data in the index allows dropping chunks irrelevant to queried time ranges without accessing them directly.  
+
+```
+┌─────────────────────────────────────────────────────────┐
+│ len <varint>                                            │
+├─────────────────────────────────────────────────────────┤
+│ ┌──────────────────┬──────────────────────────────────┐ │
+│ │                  │ ┌──────────────────────────┐     │ │
+│ │                  │ │ ref(l_i.name) <uvarint>  │     │ │
+│ │     #labels      │ ├──────────────────────────┤ ... │ │
+│ │    <uvarint>     │ │ ref(l_i.value) <uvarint> │     │ │
+│ │                  │ └──────────────────────────┘     │ │
+│ ├──────────────────┼──────────────────────────────────┤ │
+│ │                  │ ┌──────────────────────────┐     │ │
+│ │                  │ │ c_i.mint <varint>        │     │ │
+│ │                  │ ├──────────────────────────┤     │ │
+│ │      #chunks     │ │ c_i.maxt <varint>        │     │ │
+│ │     <uvarint>    │ ├──────────────────────────┤ ... │ │
+│ │                  │ │ ref(c_i.data) <uvarint>  │     │ │
+│ │                  │ └──────────────────────────┘     │ │
+│ └──────────────────┴──────────────────────────────────┘ │
+├─────────────────────────────────────────────────────────┤
+│ CRC32 <4b>                                              │
+└─────────────────────────────────────────────────────────┘
+```
+
+
+
+### Label Index
+
+The label index indexes holds lists of possible values for label names. Each label index can be a composite index over more than a single label name, which is tracked by `#names`, followed by the total number of entries.  
```

Comment:
It's not clear to me whether this is a list of label names, or label values. The structure appears to indicate names, but then what's this about composites?